examples/02-plot/image_depth.py

```diff
@@ -0,0 +1,41 @@
+"""
+Render a depth image
+~~~~~~~~~~~~~~~~~
```

Comment:
Make sure `~` matches the length of the title

Reply:
Done.